### PR TITLE
APP-4331 - Visual Testing

### PR DESCRIPTION
**Pull request by gabrielsetradantas**
Including Creevey to do Visual Testing on Storybook

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,26 @@ src/
 |   |– svg/
 ```
 
+## 🤖 Visual Testing
+
+Automated visual testing uses software to automate the process of comparing visual elements across various screen combinations to uncover visual defects.
+
+[Creevey](https://github.com/wKich/creevey/) it is an Cross-browser screenshot testing tool for Storybook with fancy UI Runner.
```
Comment (symphony-adnane) [inline README.md:45]:
```suggestion
[Creevey](https://github.com/wKich/creevey/) is a Cross-browser screenshot testing tool for Storybook with a fancy UI Runner.
```
```suggestion
[Creevey](https://github.com/wKich/creevey/) is a Cross-browser screenshot testing tool for Storybook with a fancy UI Runner.
```

File: README.md [hunk 2]
```diff
@@ -38,6 +38,26 @@ src/
...
+
+- Start storybook and then start Visual Testing Execution.
+
+```bash
+yarn start
+yarn test
+```
+And that's it.
+
+**NOTE:** After changes or new Components, the tests are gonna to fail or need to include in validation.
```
Comment (symphony-adnane) [inline README.md:55]:
```suggestion
**NOTE:** After changes or new Components, the tests will fail or need to be included in validation.
```

File: README.md [hunk 3]
```diff
@@ -38,6 +38,26 @@ src/
...
+In order to update them, you may approve them all in one command:
+```bash
```
Comment (symphony-adnane) [inline README.md:57]: what if we want to update only one ?

File: storybook-static/stories.json [hunk 4]
```diff
@@ -0,0 +1,481 @@
+{
```
Comment (symphony-adnane) [inline storybook-static/stories.json:1]: what generates this file? 
is it needed?

File: README.md (hunk 3 above)
Comment (gabrielsetradantas) [reply README.md:57]: it is the same command.

File: storybook-static/stories.json (hunk 4 above)
Comment (gabrielsetradantas) [reply storybook-static/stories.json:1]: not needed.

**Review by symphony-adnane — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (symphony-adnane) [reply README.md:57]: So if someone adds a new component
And his development breaks another component
We'll take the broken component as a reference thanks to this command?

**Review by symphony-adnane — APPROVED** [review]

**Review by symphony-jean-michael — APPROVED** [review]

**Pull request merged**